questionnaire.class.php

```diff
@@ -855,15 +857,20 @@ public function get_responses($userid=false, $groupid=0) {
             $sql = 'SELECT r.* ' .
                 'FROM {questionnaire_response} r ' .
                 $groupsql .
-                'WHERE r.questionnaireid = :questionnaireid AND r.complete = :status' . $groupcnd;
+                'WHERE r.questionnaireid = :questionnaireid' . $groupcnd;
```

Comment:
Why did you remove the 'complete' check?

Reply:
Due to the new requirement of separating the 3 response viewing modes, we have been updating this SQL query and adding some logic to check the status correctly for each view. 

![image](https://github.com/user-attachments/assets/472a64b3-1e7a-46aa-aaa6-2bbf87a1d7d1)